test/Microsoft.Framework.Internal.Test/PropertyHelperTest.cs

```diff
@@ -343,6 +343,102 @@ public void MakeFastPropertySetter_SetsPropertyValues_ForNewedProperties()
             Assert.Equal("NewedTest value", instance.PropB);
         }
 
+        [Fact]
+        public void MakeFastPropertyGetter_ReferenceType_ForNullObject_Throws()
+        {
+            // Arrange
+            var property = PropertyHelper
+                .GetProperties(typeof(BaseClass))
+                .Single(p => p.Name == nameof(BaseClass.PropA));
+
+            var accessor = PropertyHelper.MakeFastPropertyGetter(property.Property);
+
+            // Act & Assert
+            Assert.Throws<NullReferenceException>(() => accessor(null));
+        }
+
+        [Fact]
+        public void MakeFastPropertyGetter_ValueType_ForNullObject_Throws()
+        {
+            // Arrange
+            var property = PropertyHelper
+                .GetProperties(typeof(MyProperties))
+                .Single(p => p.Name == nameof(MyProperties.StringProp));
```

Comment:
Isnt this test for a value type ? 
should this be ` .Single(p => p.Name == nameof(MyProperties.IntProp))`